plugins/processors/round/round.go

```diff
@@ -0,0 +1,128 @@
+//go:generate ../../../tools/readme_config_includer/generator
+package round
+
+import (
+	_ "embed"
+	"fmt"
+	"math"
+
+	"golang.org/x/exp/constraints"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/filter"
+	"github.com/influxdata/telegraf/plugins/processors"
+)
+
+//go:embed sample.conf
+var sampleConfig string
+
+type Round struct {
+	Precision     int             `toml:"precision"`
+	IncludeFields []string        `toml:"include_fields"`
+	ExcludeFields []string        `toml:"exclude_fields"`
```

Comment:
Minor: The README shows `fields_include` and `fields_exclude`, but the struct uses `IncludeFields` and `ExcludeFields`. The TOML tags should match the documentation.